test/screenshot/lib/cloud-storage.js

```diff
@@ -16,13 +16,16 @@
 
 'use strict';
 
-const CliArgParser = require('./cli-arg-parser');
 const Duration = require('./duration');
 const GitRepo = require('./git-repo');
 const GoogleCloudStorage = require('@google-cloud/storage');
 const LocalStorage = require('./local-storage');
 const ProcessManager = require('../lib/process-manager');
-const {ExitCode} = require('../lib/constants');
+
+const {
```

Comment:
Nit: Inline these imports like the above comment.

Reply:
Done